addons/website_slides_survey/static/src/js/slides_upload.js

```diff
@@ -136,6 +152,22 @@ SlidesUpload.websiteSlidesUpload.include({
     xmlDependencies: (SlidesUpload.websiteSlidesUpload.prototype.xmlDependencies || []).concat(
         ["/website_slides_survey/static/src/xml/website_slide_upload.xml"]
     ),
+
+    /**
+     * @override
+     */
+     start: function () {
+        // Open the certification dialog if requested from query string
+        if (this.$el.attr('data-open-modal')) {
```

Comment:
Before my modifications, once the survey module had been successfully installed, the page was reloaded but "enable_slide_upload" was added in the query string, which allowed to open automatically the upload dialog.
In fact, I just took this part from the function it extends (in website_slides), but instead of simply opening the dialog I then change the state to category. 

So, maybe I misunderstood what was done originally, but when "enable_slide_upload" is in the query string, the attribute "data-open-modal" is set to true, the condition is verified and the upload dialog is opened on load.

Reply:
Yes I see that part in the code now.

Would seem a bit cleaner maybe if we add some information into "modules_to_install".
So that in addition to the module id / name / motivational message, we include a default_slide_category that would be 'certification' for the survey module.

And if you have a "default_slide_category", it could be added into the parameters when reloading the page:
```js
let redirectUrl = window.location.origin + window.location.pathname + '?enable_slide_upload';
if (this.modulesToInstallStatus.defaultSlideCategory) {
    redirectUrl += '&default_slide_category=' + this.modulesToInstallStatus.defaultSlideCategory;
}
window.location.href = redirectUrl;
```

And then when checking your parameters you could intialize the SlideUpload dialog with a default slide category.

Do you see what I mean?
Would have to be experimented a bit of course.

Reply:
Yes I see, I had more or less the same idea when modifying it and I am currently trying to do something like that.